package.json

```diff
@@ -28,7 +28,6 @@
     "**/webpack**",
     "**/babel-loader**"
   ],
-  "private": false,
```

Comment:
Above this on line 9, we have `"private": true,` in the file so I removed `pinst`.